### ISSUE TITLE
Desktop Searches No Longer Giving Credit

### ISSUE THREAD

**Issue by zacharythesharp** [jjjchens235/bing-rewards#261]
I have a script to run this everyday and I run it with
>bing-rewards.exe --exe "C:\Program Files (x86)\Google\Chrome\Application\chrome.exe"

I've had this working for months. About a week or two ago, I noticed that I no longer get credit for full desktop searches when going through the bing-rewards library. I'll get maybe 5 or 10 points, but then nothing else, even though it completes all the searches. 

Did Bing/Microsoft change something on their end to prevent scripted searches?

**Comment by heximing** [comment]
For me, I can earn desktop credits, but I cannot get mobile search credits anymore.

**Comment by kolatez** [comment]
it appears mobile search points are broken as i still get from desktop and edge too

**Comment by fexofenadine** [comment]
I think the mobile search is a separate issue, should have its own thread. Are you guys getting crashes/exceptions? Could you post the exceptions/logs to pastebin in a new issue?

**Comment by heximing** [comment]
> I think the mobile search is a separate issue, should have its own thread. Are you guys getting crashes/exceptions? Could you post the exceptions/logs to pastebin in a new issue?

No. There is no crashes/exceptions. It just says mobile search failed.

**Comment by mgrimace** [comment]
Same here as of Sept 13, 2022. The script completes successfully, but I just checked my daily search breakdown on rewards.bing.com and it shows 0/150 pc search, 0/100 mobile search and 10/20 mobile edge bonus (which I got from manually doing the this or that's and quizzes while I was there).

**Comment by heximing** [comment]
version 1.14 fixed the issue. I think this issue can be closed.
